src/main/java/sirius/db/mixing/properties/LocalTimeProperty.java

```diff
@@ -97,4 +97,10 @@ protected Object transformToJDBC(Object object) {
     public void contributeToTable(Table table) {
         table.getColumns().add(new TableColumn(this, Types.TIME));
     }
+
+    @Override
+    public String getColumnDefaultValue() {
+        // not supported
```

Comment:
```suggestion
        // If a default value is present, this will commonly be a temporal value and thus not a constant
        // (and therefore suitable for a global default value). Therefore we ignore this here. Note that
        // @DefaultValue can still be used to provide a default value to the SQL schema generator).
```